.kokoro/completeness-check.sh

```diff
@@ -13,38 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eo pipefail
+#set -eo pipefail
 
-## Get the directory of the build script
-scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+## Helper functions
 function now() { date +"%Y-%m-%d %H:%M:%S" | tr -d '\n'; }
 function msg() { println "$*" >&2; }
 function println() { printf '%s\n' "$(now) $*"; }
 
-function cleanup() {
-  rm .org-list.txt .new-list.txt .diff.txt
-}
-
-## cd to the parent directory, i.e. the root of the git repo
-cd ${scriptDir}/..
-
-## Locally install artifacts
-#mvn verify -DskipTests
-
-## Find client directory that continas flattened pom and cd into it
-clientDir=$(find -iname ".flattened-pom.xml" | cut -d"/" -f1-2)
-cd "$clientDir"
-
 ## Run dependency list completeness check
+function completenessCheck() {
+  # cd into dir containing flattened pom
+  cd "$1" || exit
```

Comment:
If you're planning on changing directories, then back, I prefer `pushd` then `popd` to go back the last directory.

Reply:
Also, I think `exit` will completely exit the script. You probably want `return <some non-zero code>` which can be dealt with by the caller.

Reply:
make sense!